Lecture2/Task1/com/playtika/UserInput.java

```diff
@@ -0,0 +1,7 @@
+package com.playtika;
+
+public class UserInput {
+    public  int minCost;
```

Comment:
I'm just wondering why you declared this fields as public? Potentially it can be acceptable for DTO "pattern", but I'm not sure that you have heard about it...